.sonarlint/sonarlint-visualstudio/CSharp/SonarLint.xml

```diff
@@ -44,7 +44,7 @@
           <Key>headerFormat</Key>
           <Value>/*
  * SonarLint for Visual Studio
- * Copyright (C) 2016-2022 SonarSource SA
+ * Copyright (C) 2016-2023 SonarSource SA
```

Comment:
Please do it in a separate PR

Reply:
I'm not sure how to revert this, could you help?

Reply:
All good 👍